config/crd/bases/consul.hashicorp.com_proxydefaults.yaml

```diff
@@ -27,85 +27,66 @@ spec:
       description: ProxyDefaults is the Schema for the proxydefaults API
```

Comment:
The yaml files are autogenerated so no need to review. The descriptions come from the struct tags so they can be reviewed there.